app/src/main/java/com/chif/headsetcontrolplus/IntroSlideSetup.java

```diff
@@ -50,13 +50,12 @@ public void onReceive(Context context, Intent intent) {
         mSignalReceived = true;
         editor.putBoolean("first", true);
         editor.apply();
-        // unregister the broadcast receiver after first successful reception
+        // Unregister the broadcast receiver after first successful reception
         getActivity().unregisterReceiver(mBroadcastReceiver);
       }
     };
 
-    // this receiver will wait to hear from the accessibility service that
-    // the headset button has worked
+    // This receiver will wait to hear from the accessibility service that the headset button has worked
```

Comment:
Nice work! However because of our current coding guidelines, we need to wrap this line.  
See https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

You can also run
```
./gradlew checkstyle
```
on Mac/Linux or
```
gradlew checkstyle
```
on Windows

Reply:
yes, I figured it out. thanks :)